src/main/java/org/candlepin/subscriptions/tally/job/CaptureSnapshotsTaskManager.java

```diff
@@ -137,13 +142,14 @@ public void updateHourlySnapshotsForAllAccounts() {
 
             Duration metricRange = appProperties.getMetricLookupRangeDuration();
             Duration prometheusLatencyDuration = appProperties.getPrometheusLatencyDuration();
+            Duration hourlyTallyOffsetMinutes = appProperties.getHourlyTallyOffset();
 
-            OffsetDateTime endTime = adjustTimeForLatency(OffsetDateTime.now().truncatedTo(ChronoUnit.HOURS),
-                prometheusLatencyDuration);
-            OffsetDateTime startTime = endTime.minus(metricRange);
+            OffsetDateTime endDateTime = adjustTimeForLatency(OffsetDateTime.now(applicationClock.getClock())
+                .minus(hourlyTallyOffsetMinutes).truncatedTo(ChronoUnit.HOURS), prometheusLatencyDuration);
```

Comment:
Change:
`
OffsetDateTime.now(applicationClock.getClock())
`
To:
`applicationClock.now()`

Could even take that as far as the following, but I'll leave that up to you.
```java
applicationClock.startOfHour(applicationClock.now().minus(hourlyTallyOffsetMinutes))
```

Reply:
Done and updated

Reply:
@dmckoske did you maybe forget to commit this file? Not seeing the change...

Reply:
That's odd.  I see the change above.

Reply:
Have a look in the files tab.  The change is there.  The quote above must not get updated with subsequent checkins or something.

Reply:
Weird. Usually it gets marked as outdated... I am good now, if you want to squash.